### DISCUSSION TITLE
Motion Event Type

### DISCUSSION THREAD

**Discussion by robitsrl** [tsightler/ring-mqtt#882] [Feature Requests]
It would be nice to know witch  kind of event make the camera recordings. People movement for example and not just movement

**Comment by tsightler** [comment]
Motion events have included `personDetected` attribute for many years.

**Comment by robitsrl** [comment]
In autonomation I do not see persondetection
![IMG_8545](https://github.com/user-attachments/assets/ad419b72-8614-4c3c-8886-c93f7ba93abd)

**Reply by tsightler** [reply]
You can see the attribute if you go to Developer Tools...States and then select your entity.  I don't know much about Home Assistant automations, certainly not the UI as I always create automations via YAML, but I believe the UI only lists default attributes for the device, not any custom ones, however, you can manually type any available attribute for the entity and match state values via a condition.  You may have to use YAML editor because I believe UI also only matches string values, not booleans.

You are generally far better to ask questions about Home Assistant automations on their forums as I'm not an expert on them in any way.
